### ISSUE TITLE
Show only deathnotices 

### ISSUE THREAD

**Issue by messioso** [akiver/cs-demo-manager#988]
### What problem is this solving

Version 3.10.1 had an option in the video sequence generation for deathnotices for all sequences. 

3.11.0 update today removed that and now I have to right click and edit each sequence and uncheck the box manually increasing time and potential errors

### Proposed solution

Re-add the checkbox for all sequences

**Comment by Bubbl666** [comment]
Could you add the command `cl_draw_only_deathnotices 0` under "Edit Sequences Settings" to override the existing configuration? I believe this will enable the HUD, though I'm not entirely sure if that's what you're aiming for.

**Comment by messioso** [comment]
Yeah I want the full HUD for all the sequences, will try that command on my next run and see how it goes.

**Comment by messioso** [comment]
Confirmed the above workaround to enter `cl_draw_only_deathnotices 0` within the "CFG" override worked. But it would be nice to have it as a UI element again like the previous version.
